pkg/ddl/storage_class_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2025 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ddl_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/pingcap/tidb/pkg/ddl"
+	"github.com/pingcap/tidb/pkg/meta/model"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBuildStorageClassSettingsFromJSON(t *testing.T) {
+	cases := []struct {
+		input    string
+		ok       bool
+		expected string
+	}{
+		{
+			input:    "",
```

Comment:
add case for `input: "ia",`, you can run with coverage to see which case is missing